include/RAJA/config.hpp.in

```diff
@@ -522,10 +533,12 @@ const int DATA_ALIGN = @RAJA_DATA_ALIGN@;
 #define RAJA_NO_SIMD
 #define RAJA_UNROLL
 #define RAJA_UNROLL_COUNT(N)
+#define RAJA_MAX_ALIGN alignof(std::max_align_t)
 
 #else
 
 #pragma message("RAJA_COMPILER unknown, using default empty macros.")
+#define RAJA_MAX_ALIGN 16
```

Comment:
RAJA_MAX_ALIGN should be consistently in the same place in the order of these per compiler defines. Here its first, but in the previous compiler it was last, and in others its in the middle.